parlai/mturk/core/react_server/dev/components/core_components.jsx

```diff
@@ -1326,6 +1364,21 @@ class ContentLayout extends React.Component {
   }
 }
 
+class StaticContentLayout extends React.Component {
+  render() {
+    let layout_style = '2-PANEL'; // Currently the only layout style is 2 panel
```

Comment:
should this be an enum?

Reply:
It should be, and eventually it will be once we support additional layouts. This is copied directly from the existing component.